shared/defs/gameObjects/unlockDefs.ts

```diff
@@ -388,4 +389,10 @@ export const UnlockDefs: Record<UnlockDefKey, UnlockDef> = {
         free: true,
         unlocks: ["outfitDarkShirt"],
     },
+    unlock_500_wins: {
+        type: "unlock",
+        name: "Champion Skin",
```

Comment:
Why is this attributed as "Champion Skin" while being named "The Pro?" Typically you would want to write the reason for acquirement here. Consider changing this as well.